solutionBuilder.js

```diff
@@ -163,13 +171,27 @@ async function processSolutions(
     branch = 'develop'
 )
 {
-    solutionFilePath = getAbsolutePath(typeof solutionFile == 'string'?solutionFile:'', '', false)
-    const merged = await mergeSolutions(
-        solutionFile,
-        directoryPath,
-        permissionMap
-    )
-    let result = {}
+    const solutionFilePath = getAbsolutePath(typeof solutionFile === 'string' ? solutionFile : '', '', false);
+
+    let solutionJson = {};
+    try {
+        solutionJson = JSON.parse(fs.readFileSync(solutionFile, 'utf8'));
+    } catch (e) {
+        console.warn('⚠️ Failed to parse root solution JSON file:', e.message);
+    }
+
+    let logoPath = null;
+    if (solutionJson?.moduleConfiguration?.logo) {
+        logoPath = path.resolve(path.dirname(solutionFilePath), solutionJson.moduleConfiguration.logo);
+    }
+
+    let themePath = null;
+    if (solutionJson?.moduleConfiguration?.theme) {
+        themePath = path.resolve(path.dirname(solutionFilePath), solutionJson.moduleConfiguration.theme);
+    }
+
+    let merged = await mergeSolutions(solutionFile, directoryPath, permissionMap);
+    let result = {};
```

Comment:
Why not doing that on merged ?

Reply:
Ok. I will check if we can do this 'on merged'. I thinkit should be possible to add into that function

Reply:
I've just checked and it could complicate the solution , especially with the scope of some variables, adding new arguments to 'mergeSolution'. I think at this stage this is the 'best' solution to manage obtaining source both theme and logo before executing 'mergeSolutions'.